src/prompts.ts

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright 2020 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import { IDP_POD_COMPAT, IDP_POD_INRUPT } from "./constants";
+import inquirer from "inquirer";
+
+const PROMPT_IDP_LIST = {
+  type: "list",
+  message: "What Solid Identity Provider do you want to register your app to?",
+  name: "identityProvider",
+  choices: [
+    { name: IDP_POD_INRUPT },
+    { name: IDP_POD_COMPAT },
+    {
+      name: "My Solid Identity provider is not on the list.",
+      value: undefined,
+    },
+  ],
+};
+
+const PROMPT_IDP_CUSTOM_INPUT = {
+  type: "input",
+  message:
+    "What is the URL of the Solid Identity Provider you want to register your app to?",
+  name: "solidIdentityProvider",
+  default: "",
+};
+
+export async function promptIdp(): Promise<string> {
+  let { identityProvider: solidIdentityProvider } = await inquirer.prompt([
+    PROMPT_IDP_LIST,
+  ]);
+  if (solidIdentityProvider === undefined) {
+    solidIdentityProvider = (await inquirer.prompt([PROMPT_IDP_CUSTOM_INPUT]))
+      .solidIdentityProvider;
+  }
+  return solidIdentityProvider;
+}
+
+const PROMPT_REGISTRATION_TYPE = {
+  type: "list",
+  message:
+    "Do you want your app to go through static, or dynamic registration?",
```

Comment:
```suggestion
    "Do you want your application to go through static, or dynamic client registration?",
```
It would be really nice if we could include a link to an Inrupt website explaining what the hell this question is asking! Can we just add a link to a dummy webpage for now (and ask Kay if we can provide a HTML page somehow later)...? Kinda awkward coming up with a link though - maybe just create a ticket for it, and point back to this comment so we can update this code once a page is there...?